lib/routes/bilibili/templates/description.art

```diff
@@ -0,0 +1,12 @@
+{{ if embed }}
+{{ if ugc }}
+<iframe width="640" height="360" src="https://player.bilibili.com/player.html?aid={{ aid }}&cid={{ cid }}&bvid={{ bvid }}" frameborder="0" allowfullscreen></iframe>
+{{ /if }}
+{{ if ogv }}
+<iframe width="640" height="360" src="https://player.bilibili.com/player.html?seasonId={{ seasonId }}&episodeId={{ episodeId }}" frameborder="0" allowfullscreen></iframe>
```

Comment:
This iframe brings back the issue that is fixed in #14517

Reply:
Done.